### PR TITLE
Issue #478

### PR DESCRIPTION
**Pull request by MimiYuchenGuo**
Fixes Issue #478

### CODE REVIEW

**Comment by MimiYuchenGuo** [comment]
My worries is now it can only check for 1 min cell count @catalamarti

File: R/validate.R [hunk 1]
```diff
@@ -626,3 +626,38 @@ validateResultArguemnt <- function(result,
   )
   validateResultArgument(result = result, validation = validation, call = call)
 }
+
+
+
+#' Title
```
Comment (catalamarti) [inline R/validate.R:632]: to populate

File: R/validate.R [hunk 2]
```diff
@@ -626,3 +626,38 @@ validateResultArguemnt <- function(result,
...
+#'
+#' @param result The suppressed result to check
+#' @param minCellCount  Minimum count of records used when suppressing
+#'
+#' @return Warning or message with check result
+#' @export
+#'
+validateSuppressedResult <- function(result, minCellCount = 5) {
```
Comment (catalamarti) [inline R/validate.R:640]: isResultSuppressed?

File: R/validate.R [hunk 3]
```diff
@@ -626,3 +626,38 @@ validateResultArguemnt <- function(result,
...
+    dplyr::select("result_id", "min_cell_count") |>
+    dplyr::mutate(.data$min_cell_count == as.integer(.data$min_cell_count))
+
+  if (all(minCellCount == unique(set$min_cell_count))) {
+    cli::cli_inform(c(
+      "v" = "The summarised result is suppressed with minCellCount = {minCellCount}."
+    ))
+    return(invisible(TRUE))
+  } else {
+    cli::cli_warn("The summarised result is suppressed because minCellCount != {minCellCount}.")
```
Comment (catalamarti) [inline R/validate.R:660]: can you provide some information if the min_cell_count is higher, smaller or what is going on?

**Review by catalamarti — CHANGES_REQUESTED** [review]
thanks @MimiYuchenGuo good job!

**Review by catalamarti — APPROVED** [review]

**Pull request merged**